### PR TITLE
WP-7986 Snowflake Connector - Allow for writebacks (send data from Symon to Snowflake) [API]

### PR DESCRIPTION
**Pull request by jayma91**
WP-7986 Snowflake Connector - Allow for writebacks (send data from Symon to Snowflake) [API]
https://varicent.atlassian.net/browse/WP-7986

### CODE REVIEW

File: target_snowflake/__init__.py [hunk 1]
```diff
@@ -430,6 +439,7 @@ def load_stream_batch(stream, records, row_count, db_sync, no_compression=False,
 
         # reset row count for the current stream
         row_count[stream] = 0
+        current_batch_size[stream] = 0
```
Comment (mjdoor) [inline target_snowflake/__init__.py:442]: Does this need to be done here if it's also being done directly in `persist_lines`, line 317?

**Comment by mjdoor** [comment]
Should we increase version in pyproject.toml?

File: target_snowflake/__init__.py (hunk 1 above)
Comment (jayma91) [reply target_snowflake/__init__.py:442]: Line 317 is for initializing current_batch_size for the stream when schema line is received. Here we are resetting the current_batch_size to 0 after loading the batch.

**Comment by jayma91** [comment]
> Should we increase version in pyproject.toml?

pipelinewise-target-snowflake's latest release is 2.2.0. Mohammed updated pyproject.toml to use version 2.2.0.1, which the tag has not yet been created.

**Review by mjdoor — APPROVED** [review]

**Review by sunil-varicent — APPROVED** [review]

**Pull request merged**